tailor-wp5.bib

```diff
@@ -222,87 +240,190 @@ @article{arxiv2022dff
+ bibsource = {dblp computer science bibliography, https://dblp.org},
+ biburl = {https://dblp.org/rec/conf/aaai/0002GFIP23.bib},
+ booktitle = {Thirty-Seventh {AAAI} Conference on Artificial Intelligence, {AAAI}
+2023},
+ doi = {10.1609/aaai.v37i6.25881},
+ editor = {Brian Williams and
+Yiling Chen and
+Jennifer Neville},
+ keywords = {WP5,A-level},
+ pages = {7227--7234},
+ publisher = {{AAAI} Press},
+ timestamp = {Mon, 04 Sep 2023 16:50:21 +0200},
+ title = {Exploiting Multiple Abstractions in Episodic {RL} via Reward Shaping},
+ url = {https://doi.org/10.1609/aaai.v37i6.25881},
+ url_paper = {https://doi.org/10.1609/aaai.v37i6.25881},
+ year = {2023}
+}
+
+@inproceedings{DBLP:conf/aaai/CimaCLP22,
+ author = {Gianluca Cima and
+Marco Console and
+Maurizio Lenzerini and
+Antonella Poggi},
+ bibsource = {dblp computer science bibliography, https://dblp.org},
+ biburl = {https://dblp.org/rec/conf/aaai/CimaCLP22.bib},
+ booktitle = {Thirty-Sixth {AAAI} Conference on Artificial Intelligence, {AAAI}
+2022},
+ doi = {10.1609/aaai.v36i5.20495},
+ keywords = {WP4,WP5,A-level},
+ pages = {5556--5563},
+ publisher = {{AAAI} Press},
+ timestamp = {Mon, 04 Sep 2023 16:50:23 +0200},
+ title = {Monotone Abstractions in Ontology-Based Data Management},
+ url = {https://doi.org/10.1609/aaai.v36i5.20495},
+ url_paper = {https://doi.org/10.1609/aaai.v36i5.20495},
+ year = {2022}
+}
+
+@inproceedings{DBLP:conf/aaai/CimaCLP23,
+ author = {Gianluca Cima and
+Marco Console and
+Maurizio Lenzerini and
+Antonella Poggi},
+ bibsource = {dblp computer science bibliography, https://dblp.org},
+ biburl = {https://dblp.org/rec/conf/aaai/CimaCLP23.bib},
+ booktitle = {Thirty-Seventh {AAAI} Conference on Artificial Intelligence, {AAAI}
+2023},
+ doi = {10.1609/aaai.v37i5.25773},
+ editor = {Brian Williams and
+Yiling Chen and
+Jennifer Neville},
+ keywords = {WP4,WP5,A-level},
+ pages = {6280--6288},
+ publisher = {{AAAI} Press},
+ timestamp = {Mon, 04 Sep 2023 16:50:23 +0200},
+ title = {Epistemic Disjunctive Datalog for Querying Knowledge Bases},
+ url = {https://doi.org/10.1609/aaai.v37i5.25773},
+ url_paper = {https://doi.org/10.1609/aaai.v37i5.25773},
+ year = {2023}
+}
+
+@inproceedings{DBLP:conf/aaai/Geffner22,
+ author = {Hector Geffner},
+ booktitle = {{AAAI}},
+ keywords = {WP5,A-level},
+ pages = {12326--12333},
+ publisher = {{AAAI} Press},
+ title = {Target Languages (vs. Inductive Biases) for Learning to Act and Plan},
+ year = {2022}
+}
+
+@inproceedings{DBLP:conf/bpm/ChiarielloPM23,
+ author = {Francesco Chiariello and
+Fabrizio Maria Maggi and
+Fabio Patrizi},
+ booktitle = {Proceedings of the Best Dissertation Award, Doctoral Consortium, and
+Demonstration {\&} Resources Forum at {BPM} 2023},
+ editor = {Dirk Fahland and
+Andr{\'{e}}s Jim{\'{e}}nez{-}Ram{\'{\i}}rez and
+Akhil Kumar and
+Jan Mendling and
+Brian T. Pentland and
+Stefanie Rinderle{-}Ma and
+Tijs Slaats and
+Johan Versendaal and
+Barbara Weber and
+Mathias Weske and
+Karolin Winter},
```

Comment:
I think that taking the condensed DBLP version is enough, so we don't need editors. But if they are already imported, we can keep the info

Reply:
You are right. I'll do that in the main tailor file for very long entries like this. I'll leave the others as they are.